src/main/java/com/google/jenkins/plugins/k8sengine/KubernetesVerifiers.java

```diff
@@ -0,0 +1,144 @@
+/*
+ * Copyright 2019 Google Inc. All Rights Reserved.
```

Comment:
`Inc. All Rights Reserved.` -> `LLC`

Reply:
That PR is here I just need to change my PR files which I'm committing shortly.